src/classes/VendorCommunicator.ts

```diff
@@ -28,7 +28,7 @@ interface EventPayload {
   projectId: string;
   versionId: number;
   metadata: MetaData;
```

Comment:
[nit] missed last time, but type should be `Metadata` (one word)

Reply:
👍